rasa/core/evaluation/markers.py

```diff
@@ -93,3 +93,20 @@ def _merge(cls, config_a: Dict, config_b: Dict) -> Dict:
             return copy_config_a
         else:
             return config_b
+
+    @classmethod
+    def _validate_config(cls, config: Dict, filename: Text = "") -> bool:
+        from jsonschema import validate
+        from jsonschema import ValidationError
+
+        try:
+            validate(config, MARKERS_SCHEMA)
+            return True
+        except ValidationError as e:
+            e.message += f". The file {filename} is invalid according to the schema."
```

Comment:
```suggestion
            e.message += f". The file {filename} is invalid according to the json schema."
```
Are you considering listing the schema somewhere in the docs too? In that case it would be great to add the URL to the docs.

Reply:
We don't yet have docs for the markers. I think I should create an issue to write the docs, and add an item to reference the docs in error messages.

Reply:
Issue created: https://github.com/RasaHQ/rasa/issues/9748

Reply:
👍🏼 Might be worth adding a tiny reminder in the issue to add the docs URL constant in error/ warning message(s) as applicable once these docs pages are available.